### PR TITLE
Update release info to v5.0.0-alpha.0

### LINKED ISSUE #2670: Prepare release info for 5.0.0-alpha.0
__What should we do?__

Prepare release info for 5.0.0-alpha.0. This should be based on the upcoming blog posts.

__Why should we do it?__

Because it's good to have it ^^

### PR DESCRIPTION
**Pull request by barmac**
Closes #2670

### CODE REVIEW

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,18 +44,25 @@ export function ReleaseInfo(props) {
...
-          <h4>Support for Camunda Cloud 1.3</h4>
-          With this release, we added support for Camunda Cloud 1.3.<br />
-          You can now assign a User Task to a specific assignee or to a group of candidates.
-          Both fields can be configured in the assignment section of the properties panel using a static value or an expression.
-          Learn more about this feature in the <a href="https://docs.camunda.io/docs/reference/bpmn-processes/user-tasks/user-tasks/#assignments">Camunda Cloud docs.</a>
+          <h4>Support for Camunda Cloud 1.4</h4>
+          With this release, we added support for Camunda Cloud 1.4.<br />
+          You can now evaluate DMN decision in a Business Rules Task.
+          Specify called decision by ID and assign the result to a variable via the properties panel.
+          Learn more about this feature in the <a href="https://docs.camunda.io/docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks/">Camunda Cloud docs on Business Rule Task.</a>
```
Comment (MaxTru) [inline client/src/plugins/version-info/ReleaseInfo.js:51]:
```suggestion
          Learn more about this feature in the <a href="https://docs.camunda.io/docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks/">Camunda Cloud docs on Business Rule Tasks.</a>
```

...cause the docs have as heading also `Business rule tasks`

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 2]
```diff
@@ -44,18 +44,25 @@ export function ReleaseInfo(props) {
...
         <li>
-          <h4>Support for Camunda Cloud 1.3</h4>
-          With this release, we added support for Camunda Cloud 1.3.<br />
-          You can now assign a User Task to a specific assignee or to a group of candidates.
-          Both fields can be configured in the assignment section of the properties panel using a static value or an expression.
-          Learn more about this feature in the <a href="https://docs.camunda.io/docs/reference/bpmn-processes/user-tasks/user-tasks/#assignments">Camunda Cloud docs.</a>
+          <h4>Support for Camunda Cloud 1.4</h4>
+          With this release, we added support for Camunda Cloud 1.4.<br />
+          You can now evaluate DMN decision in a Business Rules Task.
+          Specify called decision by ID and assign the result to a variable via the properties panel.
```
Comment (MaxTru) [inline client/src/plugins/version-info/ReleaseInfo.js:50]: Hmmm. Maybe a bit clearer: 

```suggestion
          You can now evaluate a DMN decision in a Business Rules Task.
          Specify the called decision (DMN) by its ID and assign the result to a variable via the properties panel.
```

WDYT?

**Review by MaxTru — CHANGES_REQUESTED** [review]
Currently this basically includes:

```
1. Cloud 1.4 support
2. New Prop Panel
3. New Data View / Linting errors (wrapped inside information panel, but notice that linting is not new (see forms?))
```

But in fact we have so much in this release. Maybe lets think about having a structure like this:

```
1. Cloud 1.4 support
===> Called Decision
2. Improved overall UI to become a "Visual IDE"
===> Improved Properties Panel, Welcome Page, Tab Bar, and Status Bar
3. Process Debugging for BPMN
===> For Camunda Cloud, the Modeler will now validate and show if BPMN elements are not supported in the target platform
4. Support for Collapsed Sub-Processes
```

Maybe it makes sense to do a short sync meeting to align on the general concept?

**Comment by barmac** [comment]
I think your suggestions make sense. Indeed there's so much in this release.

> 4. Support for Collapsed Sub-Processes
This is not included in this release, but will come later.

**Comment by andreasgeier** [comment]
> Maybe it makes sense to do a short sync meeting to align on the general concept?

I would suggest that @barmac creates a proposal with the three topics that @MaxTru mentioned: cloud 1.3, Visual IDE (see blog post for inspiration), debugging. Then we can discuss/review.

wdyt?

**Comment by barmac** [comment]
When I add more content, you need to scroll to see the third point:

image
image

What do you think?

**Comment by MaxTru** [comment]
I like the proposal provided by Maciej. LGTM.

**Review by andreasgeier — APPROVED** [review]

**Review by MaxTru — APPROVED** [review]

**Pull request merged**